lib/Conversation/styles.scss

```diff
@@ -116,9 +127,24 @@ $conversation-meta-text-color: $color-brand-dark !default;
 }
 
 .form__actions {
+  position: relative;
   text-align: right;
 
   .form__actions {
     margin: 0 0 0 $layout-spacing-base/2;
   }
+}
+
+.form__drop-down {
+  position: absolute;
+}
+
+.form__button {
```

Comment:
Good point. My initial though is that we should be commanding the names of our classes so using `form` should be fine.

Saying that, if `form` is a little too generic then maybe for now we could hook this up with something like `comment-form` rather than `form`? What do you think?